src/analyticsTracking/event_schemas/user/user.schema.json 

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://kf-ui-data-tracker/src/analyticsTracking/event_schemas/user/user.schema.json",
```

Comment:
Not a valid uri.
See [RFC](https://json-schema.org/draft-07/json-schema-core.html#rfc.section.8.2)